### ISSUE TITLE
"Japanese add-on with Mecab not found."

### ISSUE THREAD

**Issue by kennysong** [javdejong/nhk-pronunciation#21]
I'm using Anki 2.1.11 with the Japanese Support Plugin (version 2019-03-05) and this plugin (version 2019-01-29).

This error pops up every time I start Anki:
"NHK-Pronunciation: Mecab use requested, but Japanese add-on with Mecab not found"

I did find mecab in the Japanese Support plugin directory at addons21/⁨3918629684⁩/⁨support⁩/mecab. Maybe something's wrong with how you search for this binary?

**Comment by SpongebobSquamirez** [comment]
What operating system do you use?

**Comment by kennysong** [comment]
I'm on macOS Mojave 10.14.4.

**Comment by SpongebobSquamirez** [comment]
Okay. I updated the way Mecab was searched for a couple months ago to try to make it platform independent, but I've been waiting for @javdejong to push those changes to Ankiweb. I made those changes with Linux in mind, but if it's really platform independent then it should work for Mac too, so if you want to try the fix, simply download nhk_pronunciation.py from this repository and replace the copy in your addons folder with it, and then restart Anki.

Let me know whether it works or not.

I'll leave this issue open for other people to see, until it gets pushed to Ankiweb.
